testng-core/src/test/java/test/listeners/issue2916/ElaborateSampleTestCase.java

```diff
@@ -23,8 +23,9 @@ public void flakyTest() {
     }
   }
 
-  @Test(timeOut = 2, priority = 4)
+  @Test(timeOut = 25, priority = 4)
```

Comment:
Unexpected changes

Reply:
No it is not an unexpected change. I have just adjusted the values based on what worked from the last draft PR